autoconnect/autoconnect-common/src/protocol.rs

```diff
@@ -95,8 +93,6 @@ pub enum ServerMessage {
     Hello {
         uaid: String,
         status: u32,
-        #[serde(skip_serializing_if = "Option::is_none")]
-        use_webpush: Option<bool>,
```

Comment:
Unfortunately we need to continue sending it in the response as Desktop expects it:

https://searchfox.org/mozilla-central/rev/d6b24b15cd275874c114f2772cf6fdb0d2045847/dom/push/PushServiceWebSocket.sys.mjs#608